slycot/src/synthesis.pyf

```diff
@@ -410,13 +430,13 @@ subroutine sb10ad(job,n,m,np,ncon,nmeas,gamma,a,lda,b,ldb,c,ldc,d,ldd,ak,ldak,bk
     double precision intent(out), dimension(ncon,nmeas), depend(ncon,nmeas) :: dk
     integer intent(hide), depend(dk) :: lddk=shape(dk,0)
     double precision intent(out), dimension(2*n,2*n), depend(n) :: ac
-    integer intent(hide), depend(ac) :: ldac=shape(ac,0)
+    integer intent(hide), depend(ac) :: ldak=shape(ac,0)
```

Comment:
this (and the other changes to sb10ad) undo the fix in 42cdeb82 -- this is why the unit test fails.